themes/bootstrap3/templates/search/bulk-action-buttons.phtml

```diff
@@ -1,25 +1,25 @@
 <? if((isset($this->showBulkOptions)  && $this->showBulkOptions)
    || (isset($this->showCartControls) && $this->showCartControls)): ?>
-  <div class="bulkActionButtons hidden-print">
+  <div class="bulkActionButtons hidden-print<?if($this->idPrefix):?> form-inline<? endif; ?>">
```

Comment:
Do you think it might make more sense to include a separate attribute for form name, rather than checking idPrefix? We can rely on idPrefix based on current usage, but it's not really an intuitively logical trigger for form attribute functionality.